torchdynamo/config.py

```diff
@@ -33,6 +33,9 @@
 # disable (for a function) when cache reaches this size
 cache_size_limit = 64
 
+# specializing int/float by default
+specialize_int_float = True
```

Comment:
How would you like this to work with `capture_scalar_outputs`?

Reply:
It works well, I have [unit tests](https://github.com/pytorch/torchdynamo/pull/714/files#diff-48a9157fdd44237aa7399726b4e554f625e74de37b4c9c95acf7cdb9f2072ba4R46) covered.

Reply:
Maybe a comment on what the spec/unspec + capture/no-capture does? Not super important.